en/docs/learn/examples/proxy-service-examples/securing-proxy-services.md

```diff
@@ -12,16 +12,12 @@ Following is a sample proxy service configuration that we can used to implement
         <target>
```

Comment:
engageSec -> enableSec

Reply:
This sample demonstrates how you can use WS-Security signing and encryption with proxy services through a WS policy. 
-> 

This sample demonstrates how to use WS-Security signing and encryption with proxy services through a WS policy.

Reply:
Following is a sample proxy service configuration that we can used to implement this scenario. 
->
Following is a sample proxy service configuration that we can use to implement this scenario. S

Reply:
Please check the entire page for grammar issues.